install/installation-kubernetes.md

```diff
@@ -10,27 +10,18 @@ Before you begin installing TimescaleDB on a Kubernetes deployment, make sure yo
 * [kubectl][kubectl-install]
 * [Helm][helm-install]
 * [Kubernetes Cluster][kubernetes-install]
+* (**Optional**) Create a YAML file that specifies the values for the parameters that are provided while installing the chart. For example, you can create a new `myvalues.yaml` file. For details about the parameters you can set, see  the [Administrator Guide][admin-guide].
```

Comment:
Remove `installed` from line above (I can't edit it through GH suggestions because it's too far away from the changed section) and:

```suggestion
* Installed [kubectl][kubectl-install]
* Installed [Helm][helm-install]
* Installed [Kubernetes Cluster][kubernetes-install]
* (**Optional**) Created your own `.yaml` file if you want to use parameters other than those specified in the default `values.yaml`. You can name this file `myvalues.yaml`. For details about the parameters you can set, see  the [Administrator Guide][admin-guide].
```

Reply:
@charislam incorporated the changes you suggested.